### ISSUE TITLE
OCR0001: Training data creation for chuyik

### ISSUE THREAD

**Issue by ta4tsering** [OpenPecha/create_ocr_data#6]
Description:
Make Training Data for Chuyik

Expected Output:
- file or a desired output

Implementation Workflow:
- Image or illustration

Work items:
 - test

**Comment by tenzin3** [comment]
For the second work item "**check if the page ordering is consistent**", i split the pdf **P000010_v001_00001 - 00500.pdf** into images and 500 images were in the output. So the number count matches, as for the number in each pdf page, i manually scrolled for a minute to check if it ordering is in proper manner. and so far is does.

**Comment by ta4tsering** [comment]
yeah it sounds like they are in order, they were told to put them in order. you can go ahead and start with it. and as for getting the line images bounding box, maybe you can use transkribus to manually annotate one or two images and then use the bounding box output in xml from the transkribus to use for cropping out the line images. if you want my help with line annotation in the transkribus let me know. [trranskribus download](https://readcoop.eu/transkribus/download/). create an account and download the application and open it in your computer. we can meet tomorrow to annotate the lines.

**Comment by tenzin3** [comment]
Thank you @ta4tsering , would look into that.

**Comment by tenzin3** [comment]
To classifying the line image quality, 

### Good Image
![Image](https://github.com/OpenPecha/create_ocr_data/assets/52460417/6a803e6d-bebf-4811-b784-5e6d833a4fd8)
### Bad Image
![Image](https://github.com/OpenPecha/create_ocr_data/assets/52460417/6a16e27a-b0a8-4882-b4af-439774b6d14b)



### Black to white ration for 3500 image (Potrait)
![Image](https://github.com/OpenPecha/create_ocr_data/assets/52460417/e8d6af8d-5796-4c4d-9346-cb5e0c27b43d)

Setting threshold of 0.125 such that
    >0.125 -> Good image, 
    <=0.125 ->Bad image, 
is greate result



![Image](https://github.com/OpenPecha/create_ocr_data/assets/52460417/8f7e6934-3be7-4157-8455-dcd7c84ef6db)


### Problem 


![Image](https://github.com/OpenPecha/create_ocr_data/assets/52460417/4d5921d8-895e-4cb7-b5cf-305908b85f77)

### Reason to the problem
The reason is due to the some difference in gaps in the starting of the scanned pages, ~(8 out of 500 images) so 7*8 = 56 images
![Image](https://github.com/OpenPecha/create_ocr_data/assets/52460417/2d92475e-b693-46c4-8861-6130cfbcdd0d)

### Solution
Bigger bounding position for the line image label
